test/models/report_test.rb

```diff
@@ -16,38 +16,26 @@ class ReportTest < ActiveSupport::TestCase
   end
 
   test 'created_on' do
-    assert_equal Time.zone.today, @report_by_alice.created_on
+    @report_by_alice.created_at = DateTime.new(1993, 2, 24, 12, 30, 45)
+    assert_equal 'Wed, 24 Feb 1993'.to_date, @report_by_alice.created_on
   end
 
   test 'seve_mentions' do
     # 言及なし
-    report = {
-      user: users(:alice),
-      title: 'alice title',
-      content: 'alice content'
-    }
-    report = Report.create(report)
+    report = @alice.reports.create!(user: users(:alice), title: 'アリスの日報です', content: 'アリスです。日報を書きました。')
```

Comment:
`@alice.reports`の時点で関連が自動的に設定されます。
（このへんはRailsの基本事項なので復習しておきましょう）

```suggestion
    report = @alice.reports.create!(title: 'アリスの日報です', content: 'アリスです。日報を書きました。')
```